File: examples/after/sample.js
// Example JavaScript code with new API
const newApi = require('new-api');
const { getUserById, getUserPosts } = require('new-api');

function main() {
    const client = newApi.createClient({ key: 'secret' });
    
    // Direct call
    const user = newApi.getUserById(123);
    console.log(user);
    
    // Destructured call
    const posts = getUserPosts(123);
    
    // Client instance call
    const data = client.getUserPosts(123);
    
    return { user, posts, data };
}

main();

File: examples/after/sample.py
# Example Python code with new API
from new_api import Client, fetch_user_by_id
import new_api as api

def main():
    client = Client(api_key="secret")
    
    # Direct call
    user = new_api.fetch_user_by_id(user_id=123)
    print(user)
    
    # Call via client instance
    posts = client.list_posts_by_user(user_id=123)
    
    # Function import call
    data = fetch_user_by_id(user_id=123)
    
    # Alias call
    result = api.fetch_user_by_id(user_id=456)
    
    return user, posts, data, result

if __name__ == "__main__":
    main()
